recipe/meta.yaml

```diff
@@ -1,56 +1,149 @@
+
+{% set tests_to_ignore = "" %}
+# ignore these tests because they use pytest-cases (unavailable)
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/cases.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_construction.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_dataset_compression.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_dummy_predictions.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_fit.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_model_predict.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_pareto_front.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_performance_over_time.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_performance.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_post_fit.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_refit.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_show_models.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_automl/test_sklearn_compliance.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_ensemble_builder/test_ensemble_builder_real.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_ensemble_builder/test_ensemble_builder.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_ensemble_builder/test_manager.py" %}
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_ensemble_builder/test_run.py" %}
+# ignore on osx
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_estimators/test_estimators.py" %}  # [osx]
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_evaluation/test_evaluation.py" %}  # [osx]
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_metalearning/test_metalearning.py" %}  # [osx]
+{% set tests_to_ignore = tests_to_ignore + " --ignore=gh_src/test/test_optimizer/test_smbo.py" %}  # [osx]
 
 test:
   imports:
     - autosklearn
     - autosklearn.data
-  commands:
-    - pip check
   requires:
     - pip
+    - pytest >=4.6
+    - pytest-xdist
+    - pytest-timeout
+    - openml
+  source_files:
+    - gh_src/test
+  commands:
+    - pip check
+    # remove fixtures that use pytest-cases (unavailable)
+    - rm gh_src/test/fixtures/ensembles.py
+    - rm gh_src/test/fixtures/ensemble_building.py
+    - rm gh_src/test/fixtures/caching.py
+    - pytest -v gh_src/test {{ tests_to_ignore }} -k "not ({{ tests_to_skip }})"
+    # - pytest -v gh_src/test -k test_fit_n_jobs
```

Comment:
```suggestion
```

I will remove this comment after the long build finish and wait if there are any other changes requests